client/my-sites/plugins/use-plugins/index.ts

```diff
@@ -96,7 +96,7 @@ const usePlugins = ( {
 		isLoading: isFetchingWPORGInfinite,
 		fetchNextPage,
 	} = searchHook( wporgPluginsOptions, {
-		enabled: infinite && WPORG_CATEGORIES_BLOCKLIST.includes( category || '' ) && wporgEnabled,
+		enabled: infinite && ! WPORG_CATEGORIES_BLOCKLIST.includes( category || '' ) && wporgEnabled,
```

Comment:
@Rolilink this was the "first load problem" and what fixed the flaky e2e test. Replaces the need for this commit [`71d959e` (#63438)](https://github.com/Automattic/wp-calypso/pull/63438/commits/71d959e2e38cd9c3bdaa18e679b97deb28123577)

Reply:
My guess here will be the query was always disabled and not triggering the first fetch but calling `nextPage` was actually doing the fetching. Gonna give it some more rounds of test running and merge it